### DISCUSSION TITLE
Community Partnership for SheBuilds Code & Conduct meetup Series

### DISCUSSION THREAD

**Discussion by ConnectBhawna** [pydelhi/conference#218] [Community Partnership]
Dear PyDelhi team, 

I hope you're doing well! I’m reaching out on behalf of Shebuilds to invite PyDelhi team to support our SheBuilds Code & Connect meetup series as a Community Partner in which we organize different meetups across different cities of India , where we partner with different companies and organizations to deliver opportunities for students to learn, network & grow. These events are focused meet-ups where we talk about various products, tools, technologies, ideas and beyond.

By supporting, you’ll gain brand visibility, engage with top tech talent, and contribute to building a more inclusive tech community. We can offer benefits such as logo placements and direct engagement with attendees.

Details about event : https://www.shebuilds.tech/meetups

Fill the below Form : https://lu.ma/lef12joo

Best regards,
Shebuilds team

**Comment by satyaakam** [comment]
Hi Bhawna,

We certainly can collaborate for workshops in and around NCR region , for our volunteers  to travel to other cities is possible but who will be covering the cost etc.. is something we can discuss.

**Comment by ConnectBhawna** [comment]
We are looking to colloborate with PyDelhi Team as a community partner for Delhi & Noida region to reduce the travel cost. 
Because we are using the saving amount to provide the value to the attendees. Let me know if that works for you ?

**Comment by satyaakam** [comment]
@Daksh777 get in acall with @ConnectBhawna and see what their expectations are
